src/nvim/menu.c

```diff
@@ -698,7 +698,9 @@ static dict_T *menu_get_recursive(const vimmenu_T *menu, int modes)
         if (*menu->strings[bit] == NUL) {
           tv_dict_add_str(impl, S_LEN("rhs"), (char *)"<Nop>");
```

Comment:
BTW, it is not `<Nop>` in case of `nvim_get_keymap`, it is an empty string. Given that `:echo string("\<Nop")` echoes `'<Nop>'` and `nvim_input('<Nop>xxxxx')` neither runs any of `xxxxx` or tries to treat `<` as an operator this if() is to be removed and not the other way around (i.e. fixing `nvim_get_keymap`).

I.e. while it is nice to be displayed, it is neither consistent with `nvim_get_keymap` nor can be fed to `nvim_input()` or `feedkeys()` (latter needs something liike `feedkeys(eval('"' . substitute(string, "<", '\\<', 'g') . '"'))` to work, this is why I used `string("\<Nop>")`). Though current treatment of `<Nop>` in `nvim_input` looks like a bug.

Reply:
If I understand you correctly, this shouldn't be removed, but made into 

     tv_dict_add_str(impl, S_LEN("rhs"), (char *)"");

right?